pkg/spanconfig/spanconfigkvaccessor/testdata/system_span_configs

```diff
@@ -0,0 +1,97 @@
+# Walk through a full set of scenarios pertaining to system span configs.
+
+# Test with an empty slate.
+kvaccessor-get
+system-target {cluster}
```

Comment:
I'd recommend getting rid of this part of the grammar too. It's not adding anything that isn't represented by {source=1}, something our grammar can already represent. I also have misgivings about spanconfig.MakeClusterTarget. For one, despite the same name, it's wholly different conceptually to ptpb.MakeClusterTarget. We chatted a bit offline about this with @adityamaru, regardless of where we land, simplifying the testdata grammar itself is a good thing IMO.